### DISCUSSION TITLE
Support NextAuth.js in Bun (Elysia JS)

### DISCUSSION THREAD

**Discussion by Jojit-Thomas** [nextauthjs/next-auth#10816] [Help]
I'm building an application using Bun (Elysia JS) and I'm interested in using NextAuth.js for user authentication. Its provider support for social logins seems like a great fit for my project.

However, NextAuth.js is primarily designed for Node.js environments. I'm wondering if there's a way to leverage NextAuth.js functionalities within a Bun (Elysia JS) project.

Any insights or workarounds on potentially using NextAuth.js with Elysia JS would be greatly appreciated!

**Comment by CHINMAYJAI** [comment]
Hey i know this comment is old, but what i found is in documentation there is not any option of installing next-auth using bun 
<img width="993" height="735" alt="image" src="https://github.com/user-attachments/assets/cf86c86f-56cd-4ea9-a041-515887c920dd" />
I just randomly tries to execute command `bun add next-auth` and guess what it runs 
<img width="1108" height="332" alt="image" src="https://github.com/user-attachments/assets/70e97c94-d43d-4fc9-91b7-1eff86ffe011" />
